### ISSUE TITLE
ArrayStoreException trying to initialize CassandraSqlContext

### ISSUE THREAD

**Issue by leepa** [datastax/spark-cassandra-connector#502]
My code is simply:

```
import org.apache.spark.SparkContext
import org.apache.spark.SparkContext._
import org.apache.spark.SparkConf

import com.datastax.spark.connector._
import com.datastax.spark.connector.rdd._

import org.apache.spark.sql.cassandra.CassandraSQLContext

object SimpleApp {
  case class Test(k: String, v: Int)

  def main(args: Array[String]) {
    val conf = new SparkConf(true).setAppName("Simple Application").set("spark.cassandra.connection.host", "127.0.0.1")
    val sc = new SparkContext(conf)
    val cc = new CassandraSQLContext(sc)
    val sqlrrd = cc.sql("SELECT * FROM test.kv WHERE value > 5")
    println(sqlrrd.first)
  }
}
```

When I execute it:

```
~/Code/simplespark  spark-submit --class "SimpleApp" --master "local[4]" target/scala-2.10/fu.jar
Spark assembly has been built with Hive, including Datanucleus jars on classpath
15/01/03 19:40:30 INFO spark.SecurityManager: Changing view acls to: leepa
15/01/03 19:40:30 INFO spark.SecurityManager: Changing modify acls to: leepa
15/01/03 19:40:30 INFO spark.SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: Set(leepa); users with modify permissions: Set(leepa)
15/01/03 19:40:31 INFO slf4j.Slf4jLogger: Slf4jLogger started
15/01/03 19:40:31 INFO Remoting: Starting remoting
15/01/03 19:40:31 INFO Remoting: Remoting started; listening on addresses :[akka.tcp://sparkDriver@leepa.lan:61921]
15/01/03 19:40:31 INFO util.Utils: Successfully started service 'sparkDriver' on port 61921.
15/01/03 19:40:31 INFO spark.SparkEnv: Registering MapOutputTracker
15/01/03 19:40:31 INFO spark.SparkEnv: Registering BlockManagerMaster
15/01/03 19:40:31 INFO storage.DiskBlockManager: Created local directory at /var/folders/w1/hpz0p8_x4y51bvjdbl6zn3rr0000gn/T/spark-local-20150103194031-114a
15/01/03 19:40:31 INFO storage.MemoryStore: MemoryStore started with capacity 265.4 MB
15/01/03 19:40:31 WARN util.NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
15/01/03 19:40:32 INFO spark.HttpFileServer: HTTP File server directory is /var/folders/w1/hpz0p8_x4y51bvjdbl6zn3rr0000gn/T/spark-8baac040-ccbf-48ee-97be-0cdde29b01b6
15/01/03 19:40:32 INFO spark.HttpServer: Starting HTTP Server
15/01/03 19:40:32 INFO server.Server: jetty-8.y.z-SNAPSHOT
15/01/03 19:40:32 INFO server.AbstractConnector: Started SocketConnector@0.0.0.0:61923
15/01/03 19:40:32 INFO util.Utils: Successfully started service 'HTTP file server' on port 61923.
15/01/03 19:40:32 INFO server.Server: jetty-8.y.z-SNAPSHOT
15/01/03 19:40:32 INFO server.AbstractConnector: Started SelectChannelConnector@0.0.0.0:4040
15/01/03 19:40:32 INFO util.Utils: Successfully started service 'SparkUI' on port 4040.
15/01/03 19:40:32 INFO ui.SparkUI: Started SparkUI at http://leepa.lan:4040
15/01/03 19:40:32 INFO spark.SparkContext: Added JAR file:/Users/leepa/Code/simplespark/target/scala-2.10/fu.jar at http://192.168.1.102:61923/jars/fu.jar with timestamp 1420314032660
15/01/03 19:40:32 INFO util.AkkaUtils: Connecting to HeartbeatReceiver: akka.tcp://sparkDriver@leepa.lan:61921/user/HeartbeatReceiver
15/01/03 19:40:32 INFO netty.NettyBlockTransferService: Server created on 61924
15/01/03 19:40:32 INFO storage.BlockManagerMaster: Trying to register BlockManager
15/01/03 19:40:32 INFO storage.BlockManagerMasterActor: Registering block manager localhost:61924 with 265.4 MB RAM, BlockManagerId(<driver>, localhost, 61924)
15/01/03 19:40:32 INFO storage.BlockManagerMaster: Registered BlockManager
Exception in thread "main" java.lang.ArrayStoreException: org.apache.spark.sql.execution.SparkStrategies$CommandStrategy
	at org.apache.spark.sql.cassandra.CassandraSQLContext$$anon$1.<init>(CassandraSQLContext.scala:74)
	at org.apache.spark.sql.cassandra.CassandraSQLContext.<init>(CassandraSQLContext.scala:71)
	at SimpleApp$.main(SimpleApp.scala:16)
	at SimpleApp.main(SimpleApp.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.spark.deploy.SparkSubmit$.launch(SparkSubmit.scala:358)
	at org.apache.spark.deploy.SparkSubmit$.main(SparkSubmit.scala:75)
	at org.apache.spark.deploy.SparkSubmit.main(SparkSubmit.scala)
```

The documentation is really light around Spark SQL - so any help here would be useful.

**Comment by leepa** [comment]
On reading other and closed bugs - could this be related to Spark 1.2.0?

**Comment by bcantoni** [comment]
This project has migrated away from GitHub Issues.

Please raise your question on the Spark Connector User's list here:
https://groups.google.com/a/lists.datastax.com/forum/#!forum/spark-connector-user
